src/main/java/com/hazelcast/aws/AwsCredentials.java

```diff
@@ -0,0 +1,75 @@
+package com.hazelcast.aws;
+
+import java.util.Objects;
+
+public final class AwsCredentials {
```

Comment:
Does this need to be public? it's used just internally, right?

Reply:
done